packages/webcomponents/src/components/form/form-helpers/form-control-help-text/form-control-help-text.tsx

```diff
@@ -0,0 +1,21 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'form-control-help-text',
+})
+export class FormControlHelpText {
+  @Prop() helpText: string;
+
+  render() {
+
+    if (!this.helpText) {
+      return null;
+    }
+
+    return (
+      <small class='form-text text-muted' id='formHelpText'>
```

Comment:
I think this is going to result in some issues of having duplicate ids on the page. Can we make it unique somehow?

Reply:
Thanks for pointing that out! 

I updated this to accept `name` as a prop, which it can get from its parent form control input, and it now interpolates the `name` into the `id` of the help or error text component. 👍🏻 

Marking this comment as resolved.